tsfile/src/main/java/org/apache/iotdb/tsfile/write/writer/NativeRestorableIOWriter.java

```diff
@@ -64,14 +64,12 @@ public NativeRestorableIOWriter(File file, boolean append) throws IOException {
     if (file.exists()) {
       try (TsFileSequenceReader reader = new TsFileSequenceReader(file.getAbsolutePath(), false)) {
         if (reader.isComplete() && !append) {
-          canWrite = false;
-          out.close();
+          forceClose();
           return;
         }
         truncatedPosition = reader.selfCheck(knownSchemas, chunkGroupMetaDataList, !append);
         if (truncatedPosition == TsFileCheckStatus.COMPLETE_FILE && !append) {
-            this.canWrite = false;
-            out.close();
+          forceClose();
```

Comment:
`forceClose()` is used for test now (see the javadoc)... change the javadoc (but should declare that normal user should not use this api).

Reply:
Modified the java doc.